fenicsadapter/config.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, adapter_config_filename):
         self._coupling_mesh_name = None
         self._read_data_name = None
         self._write_data_name = None
-        self._interpolation_type = None
```

Comment:
I think this line should already have been removed in #86. Changing it now to `self._data_writing_type` gives a wrong impression. I already created another PR #88 for cleaning this up.